pkg/nitro_enclave_cpu_plugin/device_plugin.go

```diff
@@ -0,0 +1,338 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Could you please split the introduction of the `nitro_enclave_cpu_plugin` from the refactoring into its own commit. The commit messages would also benefit from a bit more color around what and why this is done.

Reply:
doen